src/pe_reports/reportlab_generator.py

```diff
@@ -0,0 +1,1839 @@
+"""Generate a P&E report using a passed data dictionary."""
+# Standard Python Libraries
+from hashlib import sha256
+import os
+
+# Third-Party Libraries
+import demoji
+import numpy as np
+from reportlab.lib import utils
+from reportlab.lib.colors import HexColor
+from reportlab.lib.pagesizes import letter
+from reportlab.lib.styles import ParagraphStyle
+from reportlab.lib.units import inch
+from reportlab.pdfbase import pdfmetrics
+from reportlab.pdfbase.ttfonts import TTFont
+from reportlab.platypus import (
+    HRFlowable,
+    Image,
+    KeepTogether,
+    ListFlowable,
+    ListItem,
+    PageBreak,
+    Paragraph,
+    Spacer,
+    Table,
+    TableStyle,
+)
+from reportlab.platypus.doctemplate import (
+    BaseDocTemplate,
+    NextPageTemplate,
+    PageTemplate,
+)
+from reportlab.platypus.flowables import BalancedColumns
+from reportlab.platypus.frames import Frame
+from reportlab.platypus.tableofcontents import TableOfContents
+
+BASE_DIR = os.path.abspath(os.path.dirname(__file__))
+
+pdfmetrics.registerFont(
+    TTFont(
+        "Franklin_Gothic_Book_Italic", BASE_DIR + "/fonts/FranklinGothicBookItalic.ttf"
+    )
+)
+pdfmetrics.registerFont(
+    TTFont("Franklin_Gothic_Book", BASE_DIR + "/fonts/FranklinGothicBook.ttf")
+)
+pdfmetrics.registerFont(
+    TTFont(
+        "Franklin_Gothic_Demi_Regular",
+        BASE_DIR + "/fonts/FranklinGothicDemiRegular.ttf",
+    )
+)
+pdfmetrics.registerFont(
+    TTFont(
+        "Franklin_Gothic_Medium_Italic",
+        BASE_DIR + "/fonts/FranklinGothicMediumItalic.ttf",
+    )
+)
+pdfmetrics.registerFont(
+    TTFont(
+        "Franklin_Gothic_Medium_Regular",
+        BASE_DIR + "/fonts/FranklinGothicMediumRegular.ttf",
+    )
+)
```

Comment:
Replace with a loop (I think this will pass linting, but I didn't check):
```suggestion
for font_name, font_filename in [
    ("Franklin_Gothic_Book", "FranklinGothicBook.ttf"),
    ("Franklin_Gothic_Book_Italic", "FranklinGothicBookItalic.ttf"),
    ("Franklin_Gothic_Demi_Regular", "FranklinGothicDemiRegular.ttf"),
    ("Franklin_Gothic_Medium_Italic", "FranklinGothicMediumItalic.ttf"),
    ("Franklin_Gothic_Medium_Regular", "FranklinGothicMediumRegular.ttf"),
    ]:
    pdfmetrics.registerFont(TTFont(font_name, BASE_DIR + "/fonts/" + font_filename))
```

Reply:
fixed linting check in this commit: [0579465](https://github.com/cisagov/pe-reports/pull/565/commits/057946512e6e9f5991ec93555f62218f75120f3c)